projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -1,44 +1,55 @@
 import { getRedirectUrl } from '@automattic/jetpack-components';
-import { getSiteFragment } from '@automattic/jetpack-shared-extension-utils';
+import { getSiteFragment, useAnalytics } from '@automattic/jetpack-shared-extension-utils';
 import { Modal, Button, CheckboxControl } from '@wordpress/components';
 import { usePrevious } from '@wordpress/compose';
 import { useSelect } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 import { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { isModalSupportedByFlow } from './util';
 import './editor.scss';
 
 export const name = 'launchpad-save-modal';
 
 export const settings = {
 	render: function LaunchpadSaveModal() {
-		const isSaving = useSelect(
+		const isSavingSite = useSelect(
 			select => select( editorStore ).isSavingNonPostEntityChanges(),
 			[]
 		);
-		const prevIsSaving = usePrevious( isSaving );
+		const prevIsSavingSite = usePrevious( isSavingSite );
 		const [ isModalOpen, setIsModalOpen ] = useState( false );
 		const [ dontShowAgain, setDontShowAgain ] = useState( false );
 		const [ isChecked, setIsChecked ] = useState( false );
 
-		const { launchpadScreenOption, siteIntentOption } = window?.Jetpack_LaunchpadSaveModal || {};
+		const { launchpadScreenOption, siteIntentOption } =
+			window?.Jetpack_Launchpad_save_modalSaveModal || {};
```

Comment:
I think that's a typo? I don't think `Jetpack_Launchpad_save_modalSaveModal` exists.

Reply:
Good catch, I'll update that